readme.md

```diff
@@ -74,6 +74,11 @@ watchedObject.a.b[0].c = true;
 // Access the original object
 watchedObject['[[target]]'].foo = false;
 // Callback isn't called
+
+// Unsubscribe
+watchedObject['[[unsubscribe]]']();
```

Comment:
Regarding https://github.com/sindresorhus/on-change/pull/42/files#r302830574, I imagine this could be:

```suggestion
watchedObject[onChange.unsubscribe]();
```

Or maybe even nicer:

```js
onChange.unsubscribe(watchedObject);
```